style.css

```diff
@@ -530,7 +533,232 @@ ul {
   z-index: 10;
 }
 
+.container-scroll {
+  width: 100%;
+  overflow-y: scroll;
+  background-color: #dc4a0cdd;
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  background-color: #fff;
+  border-radius: 30px;
+  width: 100%;
+  height: 97vh;
+  max-width: 80%;
+  padding: 15px 30px 40px 30px;
+  z-index: 100;
+  backdrop-filter: blur(0.31rem);
+  margin: 30px;
+  overflow-y: scroll;
+  box-shadow: 10px 10px 29px 0 rgba(0, 0, 0, 0.75);
+}
+
```

Comment:
- [ ] Great job on working on the card but I think it would be better if you decrease the width and height for the modal to be the same as the Figma. you can decrease the description because it is too long ✅✅

- [ ] Kindly remove the scrolling in the side for the modal  since it is not in the Figma design and it is importnat to stick with the Figma template.🔥💪🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/180222169-2659978b-e00d-4b1f-9598-63e2066c73b5.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/180222336-d677b631-92a8-44f3-bb3a-0041acb4edda.png)